ios/RNDateTimePicker.m

```diff
@@ -72,4 +72,11 @@ - (void)setDate:(NSDate *)date {
     }
 }
 
+- (void)reset
+{
+  self.minimumDate = nil;
+  self.maximumDate = nil;
+  self.date = [NSDate date];
```

Comment:
can't set this to nil - this appears to be the default when you instantiate a UIDatePicker